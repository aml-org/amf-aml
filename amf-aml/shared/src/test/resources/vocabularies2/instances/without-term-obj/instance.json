{
  "$dialect": "amf-json-schema-generated-dialect 1.0",
  "a1": {
    "b1": "something"
  }
}
