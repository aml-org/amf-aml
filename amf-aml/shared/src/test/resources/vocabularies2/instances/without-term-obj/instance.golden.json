{
  "a1": {
    "b1": "something"
  }
}
